astropy/modeling/functional_models.py

```diff
@@ -818,8 +818,7 @@ def fit_deriv(cls, x, x_0, amplitude_L, fwhm_L, fwhm_G):
         dVdx = np.sum((D/beta - 2 * (X - B) * alpha / np.square(beta)), axis=-1)
         dVdy = np.sum((C/beta - 2 * (Y - A) * alpha / np.square(beta)), axis=-1)
 
-        dyda = [constant * V / amplitude_L,
-                - constant * dVdx * 2 * sqrt_ln2 / fwhm_G,
+        dyda = [- constant * dVdx * 2 * sqrt_ln2 / fwhm_G, constant * V / amplitude_L,
```

Comment:
Could you please also put both of these on separate lines like they were before?  It's more readable that way (just in the correct order now! :)